echopype/convert/set_groups_base.py

```diff
@@ -187,22 +187,24 @@ def _parse_NMEA(self):
         return time1, msg_type, lat, lon
 
     def _beam_groups_vars(self):
```

Comment:
@emiliom I don't think this function correctly accounts for the EK80 sensor. From my understanding, there do exist EK80 files that only have the power data. Thus, these types of files would have `Beam_group1` with the description "contains complex backscatter data and other beam or channel-specific data". If you want a file that exhibits this behavior, please see `echopype/test_data/ek80_new/D20211004-T234429.raw`.

Reply:
Can you elaborate? I *think* what you're saying is that the current group description for EK80 `Beam_group1` is inadequate. If that's the case, then the fix would be to that text in `SetGroupsEK80` (note that @leewujung composed that text). This function is generic and just sets up the beam_groups_descr variable and beam_group coord variable, based on the beam group information that's passed to it.

Reply:
I think the description is fine, just how you are assigning it to the beam group is incorrect. This is a known possibility, in `1.0.yml` we have 
```
name: Beam_group1
    description: >-
      contains backscatter data (either complex samples or uncalibrated power samples)
      and other beam or channel-specific data, including split-beam angle data when they exist.
```
Note that we say either complex samples OR uncalibrated power samples

Basically, for EK80, your code should check if only one beam group exists and if only one beam group exists, then it should select the appropriate description. It seems like the issue here (at least for EK80) is that `beamgroups_possible` ties the description to a specific beam group i.e. `Beam_group1` is always for the complex backscatter data.

Reply:
Thanks. Let me chew on this. I'll get back to it mid-late afternoon, hopefully.

Reply:
Ok, I've looked at this more closely. For the sample file you listed (`test_data/ek80_new/D20211004-T234429.raw` -- thanks!), the data is written to `Beam_group1` and its description becomes the EK80 `Beam_group1` description text we have, "contains complex backscatter data and other beam or channel-specific data." I'm not sure if you were saying that that's what *should* happen but doesn't or what does happen but shouldn't, but I believe you're saying the latter. That's the intended behavior, and not something that was changed in this PR. I think @leewujung provided that EK80 beam group description text.

I still think that what you're getting at can be addressed by simply changing the description in `beamgroups_possible` that's  associated with `Beam_group1`. It sounds like something like this would be more appropriate: "contains backscatter data (either complex samples or uncalibrated power samples) and other beam or channel-specific data".

Also, just to narrow down the problem, the function `beam_groups_vars` itself would not be the source of the potential problem, or solution. It's very generic. It just writes out what's passed to it. The issue you're getting at is what's passed to it by `SetGroupsEK80.set_sonar`, which occurs here:
https://github.com/OSOceanAcoustics/echopype/blob/91dc06d2b42be14a938394f6aba9b84d3ef43f10/echopype/convert/set_groups_ek80.py#L171-L172

The code does check -- elsewhere -- if only one beam group exists; if that's the case, that group will be `Beam_group1` and the corresponding  dict from `beamgroups_possible` is passed to it. The description text for that group doesn't need to exactly specify whether the beam group is complex or power only; that *would* be nice, and we could look into it in the future, but it's not necessary. That description does need to be accurate and not misleading, though. It sounds like a change to that description along the lines of what I suggest would make it more accurate.

Reply:
> Ok, I've looked at this more closely. For the sample file you listed (`test_data/ek80_new/D20211004-T234429.raw` -- thanks!), the data is written to `Beam_group1` and its description becomes the EK80 `Beam_group1` description text we have, "contains complex backscatter data and other beam or channel-specific data." I'm not sure if you were saying that that's what _should_ happen but doesn't or what does happen but shouldn't, but I believe you're saying the latter. That's the intended behavior, and not something that was changed in this PR. I think @leewujung provided that EK80 beam group description text.

Yes, I was saying the latter. For the file you reference (and I provided), there is no complex backscatter data, so the description is wrong.  

> I still think that what you're getting at can be addressed by simply changing the description in `beamgroups_possible` that's associated with `Beam_group1`. It sounds like something like this would be more appropriate: "contains backscatter data (either complex samples or uncalibrated power samples) and other beam or channel-specific data".

Although your change to the description does fix the issue I described, it somewhat conflicts with the `Beam_group2` description: "contains backscatter power (uncalibrated) and other beam or channel-specific data, including split-beam angle data when they exist." I am sure any user could infer that `Beam_group1` corresponds to "complex samples" when two beam groups exist. However, I liked the idea of being very specific and stating whether the beam group has complex or power only data. I think this would be simple to implement and would benefit the user by providing further clarity. If you think this change to a more specific description conflicts with the intent of this PR, I understand. I am willing to take on this issue, if you do not have the time for it. 

> Also, just to narrow down the problem, the function `beam_groups_vars` itself would not be the source of the potential problem, or solution. It's very generic. It just writes out what's passed to it. The issue you're getting at is what's passed to it by `SetGroupsEK80.set_sonar`

I see where you are coming from, maybe the function `beam_groups_vars` is not the most appropriate place to change this.

Reply:
> However, I liked the idea of being very specific and stating whether the beam group has complex or power only data. I think this would be simple to implement and would benefit the user by providing further clarity. If you think this change to a more specific description conflicts with the intent of this PR, I understand. I am willing to take on this issue, if you do not have the time for it.

If you really think it'd be *very* simple to implement, go ahead; otherwise my vote would be for leaving it to 0.6.1. While the impact of this addition is "minor" on the scope of this PR (and it does fit its intent), I think it should be done in a different PR, mainly so we can move on with this one.

Reply:
Maybe "simple" was the wrong word. I would say it is doable haha. I am fine if you would like to leave this for 0.6.1. I think we have enough on our plate for 0.6.0. I will create a new issue for this. For this PR, we can just change the `Beam_group1` description to "contains backscatter data (either complex samples or uncalibrated power samples) and other beam or channel-specific data", as you suggested.

Reply:
> I am fine if you would like to leave this for 0.6.1. I think we have enough on our plate for 0.6.0. I will create a new issue for this.

That would be perfect, thank you. If you already had concrete ideas on how to accomplish this, I suggest you describe them at least briefly when creating the new issue, while they're fresh in your mind.

> For this PR, we can just change the Beam_group1 description to "contains backscatter data (either complex samples or uncalibrated power samples) and other beam or channel-specific data", as you suggested.

I'll go ahead and make the change now.